src/java.base/share/classes/sun/nio/fs/PollingWatchService.java

```diff
@@ -308,7 +308,7 @@ void enable(Set<? extends WatchEvent.Kind<?>> events, long period) {
                 // create the periodic task
                 Runnable thunk = new Runnable() { public void run() { poll(); }};
                 this.poller = scheduledExecutor
-                    .scheduleAtFixedRate(thunk, period, period, TimeUnit.SECONDS);
+                    .scheduleAtFixedRate(thunk, 1 /*init wait*/, period, TimeUnit.SECONDS);
```

Comment:
Maybe delete this `/*init wait*/` embedded comment and change the comment at line 308 to something like
```// create the periodic task with initialDelay set to the default sensitivity```? (The hard-coded 1 would of course be changed to the new constant.)

Reply:
Good suggestion. Thanks!